bridge/framework/django.py

```diff
@@ -62,16 +64,52 @@ def configure_staticfiles(self, platform: Platform):
                     "whitenoise.storage.CompressedManifestStaticFilesStorage"
                 )
 
+    def configure_worker(self, platform: Platform) -> None:
+        environment = build_redis_environment(platform)
+        self.framework_locals["CELERY_BROKER_URL"] = environment.url()
+        self.framework_locals["CELERY_RESULT_BACKEND"] = environment.url()
+
+    def start_local_worker(self) -> None:
+        # Confirm we are in a `runserver` command
+        if sys.argv[1] == "runserver":
+            # This will make sure the app is always imported when
+            # Django starts so that shared_task will use this app.
+            from bridge.service.celery import app  # noqa: F401
+
+            with log_task("Starting local worker", "Local worker started"):
+                try:
+                    subprocess.run(
+                        ["celery", "-A", "bridge.service.celery", "status"],
+                        check=True,
+                        stdout=subprocess.DEVNULL,
+                        stderr=subprocess.STDOUT,
+                    )
+
+                except subprocess.SubprocessError:
+                    subprocess.Popen(
+                        [
+                            "celery",
+                            "-A",
+                            "bridge.service.celery",
+                            "worker",
+                            "-l",
+                            "INFO",
+                        ],
+                        stdout=subprocess.DEVNULL,
```

Comment:
we could think about doing stuff with this output in the future, for now just silencing